Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/cmake-build-debug/\r\n/.idea/\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .gitignore	(revision dfa1bfd9bab7a2c66f0d03c9f71fede08151a4e0)
+++ .gitignore	(date 1590411731319)
@@ -1,2 +1,1 @@
 /cmake-build-debug/
-/.idea/
